src/main/webapp/dev/js/main/instructorCourseEnrollHandsontable.es6

```diff
@@ -0,0 +1,136 @@
+/* global Handsontable:false */
+
+/**
+ * @Global Variables
+ */
+const container = document.getElementById('spreadsheet');
+let columns = ['Section', 'Team', 'Name', 'Email', 'Comments'];
```

Comment:
Why does this need to be a global variable?

Reply:
Because at first, the `columns` var is used to initialize the Handsontable. And later when the user rearranges the columns this `columns` is changed and used to update the Handsontable.

Reply:
`columns` is only used _once_ in that block in which the rearrangement of columns is done. Beyond that, the usage to initialize the Handsontable can (and should) be replaced by hardcoding the desired value (`colHeaders: ['Section', 'Team', 'Name', 'Email', 'Comments']`). You could replace it with a constant in case you think it's a magic array (e.g. `const DEFAULT_HEADERS = ['Section', 'Team', 'Name', 'Email', 'Comments']`, but I still don't see why this `columns` needs to be a global variable. Is there something that I miss here?